AU2/plugins/CorePlugin.py

```diff
@@ -212,6 +231,86 @@ def __init__(self):
             )
         ]
 
+    def render_assassin_summary(self, assassin: Assassin) -> List[AttributePairTableRow]:
+        player_type = assassin.is_police and "Police" or "Player"
+        hidden = assassin.hidden and "HIDDEN" or ""
+        return [
+            ("ID", str(assassin._secret_id)),
+            ("Type", f"{hidden} {player_type}"),
+            ("Name", assassin.real_name),
+            *((f"Pseudonym {i+1}", p) for i, p in enumerate(assassin.pseudonyms) if p),

```

Comment:
I'm also concerned about adding 1 to the pseudonym indices being misleading wrt the `[PX_i]` format codes.